R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -243,7 +243,15 @@ test_that("read csv as DataFrame", {
   expect_equal(count(withoutna2), 3)
   expect_equal(count(where(withoutna2, withoutna2$make == "Dummy")), 0)
 
+  # writing csv file
+  csvPath2 <- tempfile(pattern = "csvtest2", fileext = ".csv")
+  write.df(df2, path = csvPath2, "csv", header = "true")
+  df3 <- read.df(csvPath2, "csv", header = "true")
```

Comment:
We could do this - but I was thinking we could also check if R's `read.csv` is able to read back the file correctly with headers ?